pkg/trace/obfuscate/sql.go

```diff
@@ -383,7 +381,7 @@ func (o *Obfuscator) obfuscateSQL(span *pb.Span) {
 	if span.Resource == "" {
 		return
 	}
-	oq, err := o.ObfuscateSQLString(span.Resource)
+	oq, err := o.ObfuscateSQLString(span.Resource, SQLOptions{})
```

Comment:
Should we not take into account the feature flag here?

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8556/commits/05a6ca769edad6ccd5d9b2d68c1f16f1c72bf3c4

Reply:
Update to use `WithOptions` as discussed offline: https://github.com/DataDog/datadog-agent/pull/8556/commits/ea8853ee2f513c9a3db42ebc4e171da6bc694722